auditbeat/tests/system/test_base.py

```diff
@@ -13,66 +13,72 @@ def test_start_stop(self):
         """
         Auditbeat starts and stops without error.
         """
-        self.render_config_template(
-            modules=[{
-                "name": "file_integrity",
-                "extras": {
-                    "paths": ["file.example"],
-                }
-            }],
-        )
-        proc = self.start_beat()
-        self.wait_until(lambda: self.log_contains("start running"))
-        proc.check_kill_and_wait()
-        self.assert_no_logged_warnings('"error": "failed to stat: lstat file.example: no such file or directory"')
+        dirs = [self.temp_dir("auditbeat_test")]
+        with PathCleanup(dirs):
+            self.render_config_template(
+                modules=[{
+                    "name": "file_integrity",
+                    "extras": {
+                        "paths": dirs,
+                    }
+                }],
+            )
+            proc = self.start_beat()
+            self.wait_until(lambda: self.log_contains("start running"))
+            proc.check_kill_and_wait()
+            self.assert_no_logged_warnings()
 
-        # Ensure all Beater stages are used.
-        assert self.log_contains("Setup Beat: auditbeat")
-        assert self.log_contains("auditbeat start running")
-        assert self.log_contains("auditbeat stopped")
+            # Ensure all Beater stages are used.
+            assert self.log_contains("Setup Beat: auditbeat")
+            assert self.log_contains("auditbeat start running")
+            assert self.log_contains("auditbeat stopped")
 
     @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
     def test_template(self):
         """
         Test that the template can be loaded with `setup --template`
         """
-        es = Elasticsearch([self.get_elasticsearch_url()])
+        dirs = [self.temp_dir("auditbeat_test")]
+        with PathCleanup(dirs):
+            es = Elasticsearch([self.get_elasticsearch_url()])
 
-        self.render_config_template(
-            modules=[{
-                "name": "file_integrity",
-                "extras": {
-                    "paths": ["file.example"],
-                }
-            }],
-            elasticsearch={"host": self.get_elasticsearch_url()})
-        exit_code = self.run_beat(extra_args=["setup", "--template"])
+            self.render_config_template(
+                modules=[{
+                    "name": "file_integrity",
+                    "extras": {
+                        "paths": dirs,
+                    }
+                }],
+                elasticsearch={"host": self.get_elasticsearch_url()})
+            exit_code = self.run_beat(extra_args=["setup", "--template"])
```

Comment:
`run_beat` accepts an `exit_code` param if you'd like to have it check the value for you.

Reply:
done